druid-shell/src/window.rs

```diff
@@ -492,8 +480,11 @@ impl WindowBuilder {
 
     /// Sets the initial window position in [display points], relative to the origin of the
     /// virtual screen.
+    /// For sub-windows like [tooltips] and [dropdowns], this position must be in window co-ordinates.
```

Comment:
Could you also change the previous sentence? I don't think it's correct anymore...

Reply:
Isnt it correct for app/parent windows? Their position must be relative to virtual screen right? Because they dont have a parent.

Reply:
It's true for app windows, but it doesn't make clear (IMO) that it's only for app windows. I'd suggest something more like "Sets the initial window position in display points. For windows with a parent, the position is relative to the parent. For windows without a parent, it is relative to the origin of the virtual screen. See also [set_level]."

Reply:
Sounds good. Made the change.